graphql/pupil/mutations.ts

```diff
@@ -196,4 +197,11 @@ export class MutatePupilResolver {
 
         return true;
     }
+
+    @Mutation(() => Boolean)
+    @Authorized(Role.ADMIN)
+    async pupilInvalidateScreening(@Arg('pupilScreeningId') pupilScreeningId?: number): Promise<boolean> {
+        await invalidatePupilScreening(pupilScreeningId);
```

Comment:
Would it make sense to always run `invalidateAllScreeningsOfPupil`? Looks like it doesn't hurt and it would make sure that we never have multiple valid entries.

Reply:
Yeah, I think that would be a better option; I implemented this function for the GraphQL endpoint because the issue required a way to invalidate a given pupil screening.
@lsc0498 What do you think?